content/03.results.md

```diff
@@ -217,7 +217,7 @@ These transcriptomic scores can be utilized to infer _TP53_ function in the abse
 
 We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (`EXTEND`) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher `EXTEND` scores suggest increased malignant potential.
 While we did not find that tumors with _TERT_ promoter (TERTp) mutations (N = 6) had significantly higher telomerase activity scores than non-mutated tumors (Wilcoxon p-value = 0.1196), we observed that `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
-Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@pmid:9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
+Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@pubmed:9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
```

Comment:
@jaclyn-taroni @jashapiro I can't get this one to render - it seems like pmid no longer works, so I had to change all over to `pubmed` - weird.

This one has no doi, no pmc. pubmed doesn't work and https doesn't work. @jashapiro I know you did some manual refs before - not sure if you can help with this one?

Reply:
I don't think this should be considered a blocker (so long as we are fully aware of what this _should be_) given our deadline and what else we have going on on our side.

Reply:
Yeah I can manually insert- didn't know if the manual ref was a quick thing or not.

Reply:
Although, and I have to check, if it's not numbered or placeholder numbered - I would have to renumber everything following (which I'm not going to do manually), I will just push it at the end as a manual ref in the doc.

Reply:
My thought process is that if this were to be accepted, there would almost certainly be author queries related to citations that we aren't even aware of yet!